ltm.go

```diff
@@ -1997,15 +2026,24 @@ func (b *BigIP) ModifyMonitor(name, monitorType string, config *Monitor) error {
 
 // PatchMonitor allows you to change any attribute of a monitor.
 func (b *BigIP) PatchMonitor(name, monitorType string, config *Monitor) error {
+	if len(config.Partition) > 0 {
+		name = "~" + config.Partition + "~"+name
+	}
 	return b.patch(config, uriLtm, uriMonitor, monitorType, name)
 }
 
 // AddMonitorToPool assigns the monitor, <monitor> to the given <pool>.
-func (b *BigIP) AddMonitorToPool(monitor, pool string) error {
+func (b *BigIP) AddMonitorToPool(monitor, pool string, partition string) error {
 	config := &Pool{
 		Monitor: monitor,
 	}
 
+	if len(partition) > 0 {
+		// node partition
+		config.Partition = partition
```

Comment:
same here, no possibility to without partition specified in POST.

Reply:
100% DeleteMonitor  and PatchMonitor will work with  "/partition/monitorname"  as they as well call iControlPath on the path